### PR TITLE
Fixed some issues & improved data documentation

### PR DESCRIPTION
**Pull request by MrDiamond64**
- Added mappings to packet_simple_event, packet_mob_effect, packet_interact, packet_lab_table and packet_video_stream_connect
- Fixed packet_book_edit, packet_set_default_game_type, packet_lectern_update, packet_structure_template_data_export_request, packet_client_cache_blob_status, packet_update_player_game_type and packet_filter_text_packet having the wrong bound type
- Added bound to packet_npc_dialogue, packet_create_photo, packet_update_subchunk_blocks, packet_photo_info_request, packet_lesson_progress, packet_game_test_request and packet_game_test_results so they dont show up as a DataType
- Fixed spelling mistake with execute_opening_commands (matched with reverse engineered bds enums to verify its correct)

### CODE REVIEW

File: data/bedrock/1.16.201/proto.yml [hunk 1]
```diff
@@ -55,7 +55,7 @@ packet_login:
    !bound: server
    # Protocol version (Big Endian!)
    protocol_version: i32
-   tokens: '["encapsulated", { "lengthType": "varint", "type": "LoginTokens" }]'
+   tokens: LoginTokens
```
Comment (extremeheat) [inline data/bedrock/1.16.201/proto.yml:58]: This change doesn't seem right. It may be failing CI

File: data/bedrock/1.16.201/proto.yml [hunk 2]
```diff
@@ -1960,7 +1980,7 @@ packet_update_block_properties:
 # what blobs it needs and which blobs it already has, in an ACK type system.
 packet_client_cache_blob_status:
    !id: 0x87
-   !bound: client
+   !bound: serverbound
```
Comment (extremeheat) [inline data/bedrock/1.16.201/proto.yml:1983]: should be `server`

**Review by extremeheat — COMMENTED** [review]
Nice work! I left some comments from a quick glance.

File: data/bedrock/1.16.201/proto.yml (hunk 2 above)
Comment (MrDiamond64) [reply data/bedrock/1.16.201/proto.yml:1983]: Ah yeah, my bad. I'll fix that

**Comment by andriycraft** [comment]
W

**Review by extremeheat — APPROVED** [review]

**Pull request merged**